src/Tizen.System.Resource/Interop/Interop.Resource.cs

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016 Samsung Electronics Co., Ltd All Rights Reserved
```

Comment:
Replace with `2022` of the copyright in all new files.
```suggestion
 * Copyright (c) 2022 Samsung Electronics Co., Ltd All Rights Reserved
```